uml-class/analyseclient/analyseclient.puml

```diff
@@ -52,7 +52,7 @@ package Analyse-Client{
     ' DiagrammType
     enum DiagrammType {
         RELFRQIMGAREA
-        VIEWTIMEDISTR
+        VIEWTIMEDISTR 
```

Comment:
wieso hast du hier ein Leerzeichen hinzugefügt?

Reply:
fett finger it is